infrastructure/README.md

```diff
@@ -0,0 +1,30 @@
+## Quickly Rolling Back a Deploy
+
+1. Since the latest tag is being used, we can't simply just repoint the image back to the previous release, but we can instead use Kubernetes' built in rollback feature.
+1. `kubectl rollout undo deployment.apps/web-editor-node -n production`
+
+## Upgrading the GKE Cluster
+
+1. Log into the [Google Cloud Platform console](https://console.cloud.google.com/) and go to the GKE service and click into the p5-gke-cluster.
+1. There are two components of GKE that need to be upgraded, the control plane (or cluster master node), and the cluster nodes themselves (or node pools), in that order.
```

Comment:
i feel dumb that i did not know you could do this with markdown to get the numbers to always be right even if you add another step